File: js/p1.js
function multiples(max) {
  var sum = 0;
  for (var i=1; i<max; i++) {
    if (i % 3 === 0 || i % 5 === 0) {
      sum += i;
    }
  }
  return sum;
}

console.log( multiples(1000) );

File: go/p1.go
package main

import (
  "fmt"
)

func multiples(max int) int {
  var sum int = 0

  for i := 0; i < max; i++ {
    if i % 3 == 0 || i % 5 == 0 {
      sum += i
    }
  }

  return sum
}

func main() {
  fmt.Printf("Now you have %d problems\n", multiples(1000))
}
